src/precice/impl/SolverInterfaceImpl.hpp

```diff
@@ -362,6 +365,29 @@ class SolverInterfaceImpl {
   /// Computes and maps all read data mapped to mesh with given ID.
   void mapReadDataTo(int toMeshID);
 
+  /**
+   * @brief Write vectorial data to the interface mesh
+   *
+   * The exact mapping and communication must be specified in XYZ.
```

Comment:
```suggestion
```
?

Reply:
These are changes to the ``writeData``-methods, which were probably a result of the merge with the current develop version, but I am not responsible for these.